### DISCUSSION TITLE
cannot import name 'UNICODE_EMOJI' from 'emoji' whe trying setup spacymoji

### DISCUSSION THREAD

**Discussion by stph89** [explosion/spaCy#11895] [Help: Installation]
import spacy
from spacymoji import Emoji

nlp = spacy.load("en_core_web_sm")
emoji = Emoji(nlp)
nlp.add_pipe("emoji", first=True)

ImportError: cannot import name 'UNICODE_EMOJI' from 'emoji'

**Comment by polm** [comment]
How did you install spacymoji and the emoji library? It sounds like you might have a version mismatch. I think you can resolve it by installing spacymoji in a blank environment, but if that doesn't work could you share the output of `pip freeze`?
